scripts/build_renewable_profiles.py

```diff
@@ -179,50 +177,50 @@
   reached.
 
 """
+import functools
+import logging
 import os
+import time
 
-import progressbar as pgb
+import atlite
 import geopandas as gpd
-import xarray as xr
 import numpy as np
-import functools
-import atlite
-import logging
+import progressbar as pgb
+import xarray as xr
+from _helpers import configure_logging
 from pypsa.geo import haversine
 from shapely.geometry import LineString
-import time
-
-from _helpers import configure_logging
 
 logger = logging.getLogger(__name__)
 
 # Requirement to set path to filepath for execution
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
This line shall be commented in the pushed version. Can be used only in debug mode